### DISCUSSION TITLE
来点图~

### DISCUSSION THREAD

**Discussion by swn2233** [share121/inter-knot#2416] [其他]
![96539574_p1](https://github.com/user-attachments/assets/f6fe408d-a2c2-476f-b90a-0432ed7a7925)
![113845354_p0](https://github.com/user-attachments/assets/c1f02691-0251-4fba-b03a-354c578229c9)
![__yae_miko_genshin_impact_drawn_by_qian356033__ab499fabbebf62390909b5e4a538bca7](https://github.com/user-attachments/assets/8ab31734-5a1f-4292-ac8f-d247a7d02c98)
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by nightrain2333** [comment]
摩多摩多<img src="https://bbs-static.miyoushe.com/static/2023/11/16/a6976b54880a512092a19f34c1cd3931_2328184749682335286.png" alt="铃-萌混过关" width="75" >
